lib/src/body.dart

```diff
@@ -181,11 +193,21 @@ class _ExpandableTableBodyState extends State<ExpandableTableBody>
                   duration:
                       ExpandableTableData.of(context).scrollShadowDuration,
                   controller: _restColumnsController,
-                  child: ListView(
-                    controller: _restColumnsController,
-                    physics: const ClampingScrollPhysics(),
-                    children: bodyColumns,
-                  ),
+                  child: ExpandableTableData.of(context).visibleScrollbar
```

Comment:
@RichiB20 Code boilerplate fix.

Edit from

```dart
ExpandableTableData.of(context).visibleScrollbar
                      ? ScrollConfiguration(
                          behavior: ScrollConfiguration.of(context)
                              .copyWith(scrollbars: false),
                          child: ListView(
                            controller: _restColumnsController,
                            physics: const ClampingScrollPhysics(),
                            children: bodyColumns,
                          ),
                        )
                      : ListView(
                          controller: _restColumnsController,
                          physics: const ClampingScrollPhysics(),
                          children: bodyColumns,
                        ),
```

to

```dart
  Builder(
    builder: (context) {
      Widget child = ListView(
        controller: _restColumnsController,
        physics: const ClampingScrollPhysics(),
        children: bodyColumns,
      );
      return ExpandableTableData.of(context)
              .visibleScrollbar
          ? ScrollConfiguration(
              behavior: ScrollConfiguration.of(context)
                  .copyWith(scrollbars: false),
              child: child,
            )
          : child;
    },
  ),
```

Reply:
Fix review and add visibleScrollbar property in README.md documentation.